src/search/pruning/stubborn_sets.cc

```diff
@@ -32,13 +31,7 @@ bool contain_conflicting_fact(const vector<FactPair> &facts1,
     return false;
 }
 
-StubbornSets::StubbornSets(const options::Options &opts)
-    : min_required_pruning_ratio(opts.get<double>("min_required_pruning_ratio")),
-      num_expansions_before_checking_pruning_ratio(
-          opts.get<int>("expansions_before_checking_pruning_ratio")),
-      num_pruning_calls(0),
-      is_pruning_disabled(false),
-      timer(false) {
+StubbornSets::StubbornSets() : num_operators(0) {
```

Comment:
Is this the best value? I see this was previously uninitialized. Rather than knowing we have 0 operators, it rather seems to be the case that we have an uninitialized attribute here. Perhaps an illegal value like -1 would communicate this more clearly.